homeassistant/components/ambient_station/sensor.py

```diff
@@ -83,7 +83,13 @@ def unit_of_measurement(self):
             w_m2_brightness_val = self._ambient.stations[self._mac_address][
                 ATTR_LAST_DATA
             ].get(TYPE_SOLARRADIATION)
-            self._state = round(float(w_m2_brightness_val) / 0.0079)
+
+            try:
+                self._state = round(float(w_m2_brightness_val) / 0.0079)
+            except TypeError:
+                # In rare cases, Ambient won't initially return a brightness value; if
+                # that happens, just return:
+                return
```

Comment:
Better be explicit if the value can be None.

```suggestion
 			if w_m2_brightness_val is None:
 				return

            self._state = round(float(w_m2_brightness_val) / 0.0079)
```

Do we want to set `self._state = None` to indicate that we currently don't know the brightness?